Automattic/wp-calypso
# Pro Dashboard: Add product tooltip on table columns.

As part of the hack week, one of the areas we decided to improve on the Agency Dashboard is to provide more information to users about the product offerings without leaving the Dashboard screen.

Related to 1202619025189113-as-1204821647785940

## Proposed Changes
This PR updates the Agency Dashboard site tables to include product tooltips that, when clicked, will display the product information. 

## Testing Instructions
Since these changes are made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb. Make sure to switch it back to the previous type.

* Use the Jetpack cloud live link below and go to `/dashboard.`
* On the site table, click the product tooltip. Confirm that the product lightbox shows up.
<img width="550" alt="Screen Shot 2023-06-14 at 5 55 26 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/487d15ce-642d-4082-a28c-57dabf499683">

* In the product lightbox, press `Issue License.` Confirm the page redirects to the Issue License screen with the pre-selected product.
<img width="1117" alt="Screen Shot 2023-06-14 at 5 56 36 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/7e18bdd1-eb8e-4980-9cc9-fa013419775c">


`

## Pre-merge Checklist



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1204821647785940
